test/Turn-test.js

```diff
@@ -19,7 +23,18 @@ describe('Turn', () => {
 
   it('should be an instance of Turn', () => {
     expect(turn).to.be.an.instanceof(Turn);
-  })
+  }),
+
+  it('should evaluate guess', function() {
+    expect(turn.evaluateGuess(data)).to.equal(true);
```

Comment:
It looks like `data` is the whole dataset here...from that assumption, I'm confused on how this `evaluateGuess` and other methods in this test file are working. Returning `true` based on passing the whole dataset through `evaluateGuess` seems confusing just based on the method name.